backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceType.java

```diff
@@ -23,26 +24,31 @@
 public class DeviceType extends EternalAuditedEntity {
 
   @Column(nullable = false)
+  @JsonView(PublicDeviceType.class)
   private String name;
 
   @Column(nullable = false)
+  @JsonView(PublicDeviceType.class)
   private String manufacturer;
 
   @Column(nullable = false)
+  @JsonView(PublicDeviceType.class)
   private String model;
 
   @JoinTable(
       name = "device_specimen_type",
       joinColumns = @JoinColumn(name = "device_type_id"),
       inverseJoinColumns = @JoinColumn(name = "specimen_type_id"))
   @OneToMany(fetch = FetchType.LAZY)
+  @JsonView(PublicDeviceType.class)
   private List<SpecimenType> swabTypes;
 
   @Column(nullable = false)
+  @JsonView(PublicDeviceType.class)
   private int testLength;
 
-  @JsonIgnore
```

Comment:
@emyl3 Have you looked into our concerns that removing these ignore tags will cause an issue?

Reply:
And I apologize because I just now realized I never added that detail to the ticket:

https://github.com/CDCgov/prime-simplereport/issues/7492#issuecomment-2532572123

Reply:
I had previously deployed [this branch](https://github.com/CDCgov/prime-simplereport/compare/main...elisa/audit-logging-duration) both with and without the `@JsonIgnore` -- is this what you were thinking in terms of adding a timer and logging to see if there's a difference with and without the `@JsonIgnore`

This is with the JsonIgnore tag
![Screenshot 2024-11-15 at 12 21 54](https://github.com/user-attachments/assets/eb488886-164c-4893-a738-c5a01bb5a12c)

This is without
<img width="452" alt="Screenshot 2024-11-15 at 13 40 56" src="https://github.com/user-attachments/assets/427d1158-359e-4a06-94b5-49aff0bc6dc7" />

Reply:
Let me know if this is not what you had in mind and I can adjust the testing plan and retest 😅

Reply:
I talked to @mehansen about this a bit and I loved her idea of just trying to replicate the original issue as a means of confirming that its no longer a concern.  But again, the lack of an audit log in the db might make that a futile effort.

Reply:
After discussing with @DanielSass @mehansen and @mpbrown after standup, we will deploy this after the cert renewal.

Once merged in, I will keep an eye on the following:
- [prod Azure dashboard](https://portal.azure.com/#@cdc.onmicrosoft.com/dashboard/arm/subscriptions/7d1e3999-6577-4cd5-b296-f518e5c8e677/resourcegroups/prime-simple-report-prod/providers/microsoft.portal/dashboards/dc2c3ec7-c167-4860-a0eb-6d8bdbde28e0)
- app insights page
- DB request avg times

If there are any slowdowns noticeable from Azure following the deploy, we will revert this PR and see if there are ways we can investigate this further.

In the meantime, I will look into if there's a way to view DB request times in Azure or check-in with Shanice after cert renewal.

Reply:
I'm seeing the our "Azure Database for PostgreSQL - Flexible Server" view for our prod DB and there is a way to view both the [Read and Write Throughput Bytes/Sec (Avg)](https://portal.azure.com/#@cdc.onmicrosoft.com/resource/subscriptions/7d1e3999-6577-4cd5-b296-f518e5c8e677/resourceGroups/prime-simple-report-prod/providers/Microsoft.DBforPostgreSQL/flexibleServers/simple-report-prod-flexible-db/metrics) -- Would this be sufficient? 🤔

Reply:
@shanice-skylight Are you familiar with any Azure tools to monitor avg DB request times? I saw [there's this](https://learn.microsoft.com/en-us/azure/azure-sql/database/query-performance-insight-use?view=azuresql) but we don't seem to have that enabled. 🤔